arm_robots/src/arm_robots/cartesian.py

```diff
@@ -78,12 +79,11 @@ def __init__(self, tf_buffer, motion_status_listeners, motion_command_publisher,
         self._check_joint_limits = True
         self._start_violation = 0
 
-        self.active_arm = ArmSide.LEFT
+        self.active_arm = 0
```

Comment:
why not just delete everything related to "active_arm" if it's going to be hard-coded to 0?

Reply:
It's not hard coded to 0; for Victor the active arm can be 0 or 1. The controller has a variable number of arms.